x/liquidstake/keeper/liquidstake.go

```diff
@@ -226,7 +226,6 @@ func (k Keeper) DelegateWithCap(
 	}
 	res, err := handler(ctx, msgDelegate)
 	if err != nil {
-		k.Logger(ctx).Error("failed to execute delegate msg,", "msg", msgDelegate.String(), "err", err)
```

Comment:
Why removing this error ?

Reply:
Can you give me some examples? Most of the cases we needed error logging in `x/liquidstakeibc` were in the `BeginBlock` or IBC callback functions which we can't return error or can't track them from the transactions.

Reply:
As long as this is logged somewhere (even if it is up the call stack) then it is fine.

Let's just make sure the logging happens as this is a very sensible point.

Reply:
keep this, we would be notified when our valset cannot accept delegations ( because of any caps )

Reply:
Ok, got it!